### DISCUSSION TITLE
RabbitMQ support

### DISCUSSION THREAD

**Discussion by humanamburu** [serverless/serverless#9995] [Q&A] [answered]
Hello,

According to [docs](https://www.serverless.com/framework/docs/providers/aws/events/activemq) there is `activemq` event supported. 

Are there any plans or ways to use serverless with the RabbitMQ deployed on Amazon MQ?
Thanks in advance.

**Comment by mnapoli** [comment, marked as answer]
Hi, check out https://github.com/serverless/serverless/issues/9924

**Comment by humanamburu** [comment]
Thanks!
